src/components/Navbar/MenuItems.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import * as FaIcons from "react-icons/fa";
+
+export const MenuItems = [
+  {
+    title: "Moje konto",
+    url: "#",
```

Comment:
As far as I know, you already have routing for most pages and even if some is missing, you could ask your team leader to answer you under what URL would be a certain page available to already set it.